rollup-operator/src/server/operator.js

```diff
@@ -58,9 +66,14 @@ let pool;
 
 (async () => {
     let info;
+
+    // Parse client command arguments
+    const passString = (argv.passphrase) ? argv.passphrase : "nopassphrase";
+    const pathEnvFile = (argv.pathconfig) ? argv.pathconfig : pathEnvFileDefault;
+    const clearFlag = (argv.clear) ? argv.clear : false;
+
     // load environment data
-    const pathEnvironmentFile = `${__dirname}/config.env`;
-    require("dotenv").config({ path: pathEnvironmentFile });
+    require("dotenv").config({ path: pathEnvFile });
```

Comment:
i think it would be better: 
```javascript
  if (fs.existsSync(pathEnvironmentFile)) {
        require("dotenv").config({ path: pathEnvironmentFile });
    } 

 if (!process.env.OPERATOR_MODE && !process.env.LOG_LEVEL && !process.env.CONFIG_SYNCH &&  
        !process.env.CONFIG_POOL && !process.env.GAS_MULTIPLIER && ! process.env.GAS_LIMIT 
       && !process.env.URL_SERVER_PROOF && !process.env.OPERATOR_PORT_EXTERNAL)
    {
        throw new Error("There's need a config.env or enviroment variables to proceed, in README.md there's further information");
    }
```

Reply:
Implemented in https://github.com/iden3/rollup/pull/84/commits/04245b1892d1e47ff80be07bb4a8379eca5df3d5

Reply:
great ^^ :dancer: